### DISCUSSION TITLE
Selecting My Custom character In Debug Menu Crashes my Game

### DISCUSSION THREAD

**Discussion by caserreal** [KadeArchive/Kade-Engine#233] [Q&A]
Now, I already submitted an issue earlier, but this shouldn't be as difficult. I have my Character sprite sheet, Xml file, and code in the characters.hx file, but when i try to test it in the debug menu, my game crashes. 
![image](https://user-images.githubusercontent.com/82672666/115084459-f37a4a80-9ed6-11eb-88ec-465a7135f2ac.png)
![image](https://user-images.githubusercontent.com/82672666/115084760-4f44d380-9ed7-11eb-91b4-dd6dace260bf.png)
If there's any solution you can recommend, please let me know as soon as possible!

**Comment by theDetourist** [comment]
What does it show on your console after before it crashes?

**Reply by caserreal** [reply]
Nothing

**Reply by caserreal** [reply]
It just crashes
